test/e2e/app-dir/app-static/app-static.test.ts

```diff
@@ -1511,6 +1511,7 @@ createNextDescribe(
               ],
               "initialHeaders": {
                 "set-cookie": "theme=light; Path=/,my_company=ACME; Path=/",
+                "x-middleware-set-cookie": "theme=light; Path=/; my_company=ACME; Path=/",
```

Comment:
We might want to filter these from `initialHeaders` since it already has `set-cookie` and a static page can't read cookies